### ISSUE TITLE
Integrate Nextcloud Workflow

### ISSUE THREAD

**Issue by stefan-niedermann** [nextcloud/notes#376]
On the NC conference Nextcloud Workflow has been introduced.

As soon as it is ready, we should enable an "create note" action :)

**Comment by sunjam** [comment]
For added context, [here is the announcement](https://help.nextcloud.com/t/nextcloud-flow-makes-it-easy-to-automate-actions-and-workflows/60007).
* [Pull request for the Workflow app overhaul](https://github.com/nextcloud/workflow_script/pull/22)
* In order to make use of workflows, tag support within this app could be implemented as mentioned in issue #299 
* How this functionality will be added might be assisted by the app navigation discussion #364 
* [This discussion of system wide tagging across all major apps](https://github.com/nextcloud/server/issues/13330) could greatly benefit and simplify the end user execution of workflows.

**Comment by korelstar** [comment]
Since we require NC20, now, we can implement this finally. However, it looks like there is still no real documentation for this. Is the flow project still alive? Which apps implemented this approach?

**Comment by marcelklehr** [comment]
Yep, the docs for this suck. This is how I did it: https://github.com/nextcloud/bookmarks/tree/master/lib/Flow & https://github.com/nextcloud/bookmarks/blob/master/src/flow.js
